code B.html

```diff
@@ -43,10 +43,29 @@
     <div class="center2"><b>Code B</b></div>
     <div class="center"><img style="width:250px" src="STEM Pictures/STEM logo1.png" alt="STEM_Logo"></div>
   </div>
-  <!--End of Menu section--)
+  <!--End of Menu section-->
 
 <h3>Our Program</h3>
-<body><strong> Code B is one of many opertunties that allows future programers or games design to build a path for themselves or to even sharpen their skills.  BGHS STEM Alumni founded the program and are working diligently to create informative tutorials on how to design games. You will learn step by step on how to design and code games with informative tutorials lasting 5-15 minutes. Learn what it takes to make a professional or fun game with NASA software engineer Julian Vu and experience the fun process of game design</strong>
+  <style>
+p{
+  border: 7px solid black;
+  margin: 10px
+}
+img{
+border:5px solid black;
+margin:10px
+}
+</style>
+<p><strong> Code B is one of many opertunties that allows future programers or games design to build a path for themselves or to even sharpen their skills.  BGHS STEM Alumni founded the program and are working diligently to create informative tutorials on how to design games. You will learn step by step on how to design and code games with informative tutorials lasting 5-15 minutes. Learn what it takes to make a professional or fun game with NASA software engineer Julian Vu and experience the fun process of game design</strong>
 
+
+</p>
+<h4> <strong> Developers of Code B </h4> </strong>
+<li> 1. Julian Vu</li>
+<img src="img_https://scontent-lax3-1.xx.fbcdn.net/v/t1.15752-0/p280x280/65872256_473267583217647_3296007020196397056_n.jpg?_nc_cat=100&_nc_oc=AQm46LiAbXpfAl2ADURcKh2PUAd8wfbX0QFCJgX17lEy8ymtIf-d1Oak4FL4PjzVv9q_4uZwaM_Nxdg4XGOnSdAJ&_nc_ht=scontent-lax3-1.xx&oh=7a01ffb66dca6e39aee9fc5996bcfc22&oe=5DA1F1E0" alt="Julian Vu" width="200" height="200">
+<li> 2. Danny Vo</li>
+<img src="img_https://scontent-lax3-1.xx.fbcdn.net/v/t1.15752-0/p280x280/65166863_895534470814523_8487793827250176000_n.png?_nc_cat=105&_nc_oc=AQkKHhRjk0WRWp4QEGr8G0siZgRysRCEV7YmUkkRmZ8iRO-Ey7mwMxWaIaDqeADfyE1h0QFQYSsRNF9rAWYKTFmz&_nc_ht=scontent-lax3-1.xx&oh=5567750a1035242cd552e3a32f659306&oe=5DA12B55" alt="Danny Vo" width="200" height="200">
+<li> 3. Duy Pham</li>
+<img src="img_chania.jpg" alt="Duy Pham" width="200" height="200">
```

Comment:
Use <ol> tag for a number-list, with each item starts with <li> tag 
EX: 
<ol>
     <li>...<li>
     <li>...<li>
<ol>